ui/components/app/modals/account-details-modal/account-details-modal.component.js

```diff
@@ -32,9 +33,14 @@ export default class AccountDetailsModal extends Component {
       setAccountLabel,
       keyrings,
       rpcPrefs,
+      accounts,
     } = this.props;
     const { name, address } = selectedIdentity;
 
+    const accountsNames = Object.values(accounts)
+      .map((item) => item.name)
+      .filter((itemName) => itemName !== name);
```

Comment:
I propose that in the `map` and `filter`, we cast account names to lower case, so that we don't allow `Account 1` and `account 1`

Reply:
We agreed with @danjm to support case-sensitive account names. @danjm, do you want us to implement it in the other way as suggested above?

Reply:
Yeah, I was thinking that it is okay if a user has both `Account 1` and `account 1`. These are only labels, so there is no risk of data corruption. And the user can edit at any time. Some users may want to take advantage of differences in casing... or may already do that intentionally.

Basically: I think there is not risk in allowing this, and if there is any possibility that someone wants to name one account `Triple AAA` and another `triple aaa` or whatever, then I thought it made sense just to stick with a strict comparison.

Reply:
Fixed in c71199a60e2949308b7659f613fe02a48f46d79d.